deps/GraphBLAS/Source/GxB_Matrix_tuple_iter.c

```diff
@@ -13,59 +24,118 @@ static inline void _EmptyIterator
 	iter->nnz_idx = 0 ;
 }
 
-GrB_Info GxB_MatrixTupleIter_init
+static inline GrB_Info _MatrixTupleIter_init
```

Comment:
Why did you decided to mark this function as `inline`?

Reply:
Cause it faster to use it, and it's being used by reuse function which being called many times